docs/content/installation/pgo-client.md

```diff
@@ -161,11 +161,11 @@ pod. You can run single commands with the kubectl or oc command line tools
 or multiple commands by exec'ing into the pod with bash.
 
 ```
-kubectl exec -it -n pgo <pgo-client-deployment-name> -c "pgo version"
+kubectl exec -it -n pgo pgo-client-<pod-id> -- pgo version
```

Comment:
This can actually work by referring to the deploy:

```sh
kubectl exec -it -n pgo deploy/pgo-client -- pgo version
```

Reply:
I was not aware. thats awesome! I made that change but it ended up being two commits. I can go in and squash them in the cli unless there is a way to do that through GitHub.

Reply:
I can squash at merge.